src/utils/index.ts

```diff
@@ -2,7 +2,7 @@ export * from './artifact-v2';
 export * from './available-rpc';
 export * from './compare';
 export * from './fallback-provider';
-export * from './error';
+export * from './errors/error';
```

Comment:
Perhaps this is an "internal module" so changing the path is insignificant, but if something is importing the error formatter directly, this is a breaking change